db/models.py

```diff
@@ -14,3 +14,40 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+    actors = models.ManyToManyField(Actor, related_name="movies")
+    genres = models.ManyToManyField(Genre, related_name="movies")
+
+    def __str__(self) -> str:
+        return self.title
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField(default=0)
+    seats_in_row = models.IntegerField(default=0)
+
+    def __str__(self) -> str:
+        return self.name
+
+    @property
+    def capacity(self) -> int:
+        return self.rows * self.seats_in_row
+
+
+class MovieSession(models.Model):
+    show_time = models.DateTimeField()
+    cinema_hall = models.ForeignKey(CinemaHall,
+                                    related_name="movie_sessions",
+                                    on_delete=models.CASCADE)
+    movie = models.ForeignKey(Movie,
+                              related_name="movie_sessions",
+                              on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        return (f"{self.movie.title}"
+                f"{self.show_time.strftime("%Y-%m-%d %H:%M:%S")}")
```

Comment:
There is a syntax error in the `__str__` method of the `MovieSession` class. The `strftime` method call should use single quotes around the format string, like `'%Y-%m-%d %H:%M:%S'`, or escape the double quotes inside the format string to avoid errors.